x/slashing/types/expected_keepers.go

```diff
@@ -53,6 +53,8 @@ type StakingKeeper interface {
 
 	// IsValidatorJailed returns if the validator is jailed.
 	IsValidatorJailed(ctx context.Context, addr sdk.ConsAddress) (bool, error)
+
+	GetNewToOldConsKeyMap(context.Context, sdk.ConsAddress) (sdk.ConsAddress, error)
```

Comment:
nit: I'd call this `GetConsKeyRotationMapping`, a bit more verbose but clearer IMO

Reply:
https://github.com/cosmos/cosmos-sdk/pull/18236#discussion_r1423706568